docs/src/man/comparisons.md

```diff
@@ -1,77 +1,204 @@
+| Column indexing by label   | `df.loc[:, 'x']`       | `df[:, :x]`                        |
+| Column slicing by label    | `df.loc[:, ['x','z']]` | `df[:, [:x, :z]]`                  |
+|                            | `df.loc[:, 'x':'z']`   | `df[:, Between(:x, :z)]`           |
+| Mixed indexing             | `df.loc['c'][1]`       | `df[findfirst(==('c'), df.id), 2]` |
+
+Note that Julia uses 1-based indexing, inclusive on both ends. A special keyword `end` can be used to
+indicate the last index.
+
+In the DataFrames.jl examples, the `findfirst` function is used to find the first match and return the result
+as a single `DataFrameRow` object. In the case that `id` is not unique, you can use the `findall` function
+or boolean indexing instead. It would then return a `DataFrame` object containing all matched rows. The following
+two lines of code are functionally equivalent:
+
+```julia
+df[findfirst(==('c'), df.id), :]
+df[df.id .== 'c', :]
+```
+
+### Common operations
+
+| Operation                | pandas                                                | DataFrames.jl                           |
+| :----------------------- | :---------------------------------------------------- | :-------------------------------------- |
+| Reduce multiple values   | `df['z'].mean(skipna = False)`                        | `mean(df.z)`                            |
+|                          | `df['z'].mean()`                                      | `mean(skipmissing(df.z))`               |
+|                          | `df[['z']].agg(['mean'])`                             | `combine(df, :z => mean ∘ skipmissing)` |
+| Add new columns          | `df.assign(x_mean = df['x'].mean())`                  | `transform(df, :x => mean => :x_mean)`  |
+| Rename columns           | `df.rename(columns = {'x': 'x_new'})`                 | `rename(df, :x => :x_new)`              |
+| Pick & transform columns | `df.assign(x_mean = df['x'].mean())[['x_mean', 'y']]` | `select(df, :x => mean, :y)`            |
+| Sort rows                | `df.sort_values(by = ['x'])`                          | `sort(df, :x)`                          |
+
+Note that Julia propagates `missing` data by default for safety reasons. The `skipmissing` function
+can be used to remove missing data. See more details at the [Additional Differences](@ref) section below.
+
+### Grouping data and aggregation
+
+| Operation                    | pandas                                                                                    | DataFrames.jl                               |
+| :--------------------------- | :---------------------------------------------------------------------------------------- | :------------------------------------------ |
+| Aggregate by groups          | `df.groupby('grp')['x'].mean().reset_index()`                                             | `combine(groupby(df, :grp), :x => mean)`    |
```

Comment:
in pandas the resulting column has name `x`, while in DataFrames.jl it has name `x_mean`. I would add examples how to get both results. Then the first one would be:

```suggestion
| Aggregate by groups          | `df.groupby('grp')['x'].mean().reset_index()`                                             | `combine(groupby(df, :grp), :x => mean => :x)`    |
```

I do not know pandas enough to say how to get the second one (i.e. rename column to `x_mean` in the process). Could you please suggest this? Thank you!

Reply:
When I went through these examples, I realized that pandas and DataFrames are quite different. The default of column naming after aggregation is one of them. Because these differences are everywhere, I think it would be better to focus on what actually gets done rather than making them match exactly. Otherwise, it will uglify either the pandas syntax or the DataFrames syntax.

The remedy is to point out these differences explicitly. This particular issue is explained in the 2nd note under "Additional Difference" section. Maybe that's too "far away". I can move that note to be directly under this section.